### PR TITLE
Add Kibana Dashboard as an iframe

### PR DESCRIPTION
**Pull request by jujaga**
# Description

This PR embeds a Proof of Concept Kibana dashboard exposed through a reverse-proxy.
At this time, the URL is written in as a default.json config entry. Once things settle down more with infrastructure we will need to revisit and have this value be supplied at the Openshift configmap level.

[SHOWCASE-398](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-398)

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wfezkf-tools.pathfinder.gov.bc.ca/dashboard?id=document-generation-showcase-pr-35

**Comment by bcgov-nr-csst** [comment]
SonarQube Analysis: https://sonarqube-wfezkf-tools.pathfinder.gov.bc.ca/dashboard?id=document-generation-showcase-pr-35

File: app/frontend/src/views/Dashboard.vue [hunk 1]
```diff
@@ -1,18 +1,47 @@
...
+  computed: {
+    dashboardUrl() {
+      return this.$configService.get('dashboardUrl');
+    }
+  }
 };
 </script>
+
+<style lang="scss" scoped>
+.dashboard-container {
```
Comment (parc-jason) [inline app/frontend/src/views/Dashboard.vue]: do we need the iframe resizer js?

Comment (jujaga) [reply app/frontend/src/views/Dashboard.vue]: I actually tried to integrate that in but wasn't able to get that to work with the vue.js framework (even their example on how to add it as a vue directive kept failing). I did the media-query approach as a makeshift workaround for the moment to make sure it roughly fits on the page.

Comment (loneil) [reply app/frontend/src/views/Dashboard.vue]: Does iframeresizer work on a Kibana dash?
It has to have a equivalent "client JS" in the target of the iframe that the resizer communicates to. (Metabase provides this in its dash)

**Pull request merged**